api/tests/routes/backoffice/gdpr_user_extract_data_test.py

```diff
@@ -0,0 +1,78 @@
+import datetime
+
+from flask import url_for
+import pytest
+
+from pcapi.core.permissions import models as perm_models
+from pcapi.core.testing import assert_num_queries
+from pcapi.core.testing import override_features
+from pcapi.core.users import factories as users_factories
+from pcapi.routes.backoffice.filters import format_date
+
+from .helpers import html_parser
+from .helpers.get import GetEndpointHelper
+
+
+pytestmark = [
+    pytest.mark.usefixtures("db_session"),
+    pytest.mark.backoffice,
+]
+
+
+@pytest.fixture(scope="function", name="list_of_gdpr_user_extract_data")
+def gdpr_user_extract_data_fixture() -> tuple:
+    gdpr_1 = users_factories.GdprUserDataExtractBeneficiaryFactory()
+    gdpr_2 = users_factories.GdprUserDataExtractBeneficiaryFactory(dateProcessed=datetime.datetime.utcnow())
+    gdpr_3 = users_factories.GdprUserDataExtractBeneficiaryFactory()
+    gdpr_4 = users_factories.GdprUserDataExtractBeneficiaryFactory()
+    gdpr_5 = users_factories.GdprUserDataExtractBeneficiaryFactory(
+        dateCreated=datetime.datetime.utcnow() - datetime.timedelta(days=8)
+    )
+    return gdpr_5, gdpr_4, gdpr_3, gdpr_2, gdpr_1
+
+
+class ListGdprUserExtractDataTest(GetEndpointHelper):
+    endpoint = "backoffice_web.gdpr_extract.list_gdpr_user_data_extract"
+    needed_permission = perm_models.Permissions.EXTRACT_PUBLIC_ACCOUNT
+
+    # Use assert_num_queries() instead of assert_no_duplicated_queries() which does not detect one extra query caused
+    # by a field added in the jinja template.
+    # - fetch session (1 query)
+    # - fetch user (1 query)
+    expected_num_queries_when_no_query = 2
+    # - fetch gdpr_data_extract
+    # - fetch temporary FF WIP_BENEFICIARY_EXTRACT_TOOL
+    expected_num_queries = expected_num_queries_when_no_query + 2
```

Comment:
```suggestion
    # 1 fetch session (1 query)
    # 2 fetch user (1 query)
    # 3 fetch gdpr_data_extract
    # 4 fetch temporary FF WIP_BENEFICIARY_EXTRACT_TOOL
    expected_num_queries = 4
```

`expected_num_queries_when_no_query` ne me semble pas utilisé donc on peut simplifier